modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -281,6 +302,63 @@ typedef struct indigo_core_gentable_ops {
         void *table_priv, void *entry_priv, of_list_bsn_tlv_t *key,
         of_desc_str_t err_txt);
 
+    /**
+     * @brief Add an entry
+     * @param op_ctx operation context. These data will be used when resume
+     *               function is called in the async operation. 
```

Comment:
"Opaque context to be passed back to state manager when resume function is called after async operation completes"

also same english should be repeated for modify4 and delete4